clang/lib/Sema/SemaChecking.cpp

```diff
@@ -8497,7 +8498,8 @@ ExprResult Sema::BuildAtomicExpr(SourceRange CallRange, SourceRange ExprRange,
         << 0 << AdjustedNumArgs << static_cast<unsigned>(Args.size())
         << /*is non object*/ 0 << ExprRange;
     return ExprError();
-  } else if (Args.size() > AdjustedNumArgs) {
+  }
+  if (Args.size() > AdjustedNumArgs) {
```

Comment:
The change is correct per our coding standard, but is unrelated to the changes in your patch. I'd say you can land this specific change as an NFC commit if you'd like, but otherwise it should be backed out of this PR.